front/app/containers/LoginPage/SignUpPage.tsx

```diff
@@ -76,7 +76,10 @@ class SignUpPage extends React.Component<SignUpPageProps, SignUpPageState> {
 
   handleSignUp = (signUp: SignUpMutationFn) => () => {
     if (this.state.form.password === this.state.form.passwordConfirmation) {
-      const input = omit(['passwordConfirmation','oAuthToken'], this.state.form);
+      const input = omit(
```

Comment:
this just seems to be prettier getting applied. i removed a `console.log` call from this file.